rts/lra/coordinator/src/test/java/io/narayana/lra/coordinator/tools/osb/mbean/ObjStoreBrowserLRATest.java

```diff
@@ -90,7 +90,8 @@ public void lraMBean() throws Exception {
 
     @Test
     public void lraMBeanRemoval() throws Exception {
-        LongRunningAction lra = new LongRunningAction(new Uid());
+        String lraUrl = "http://localhost:8080/lra";
```

Comment:
Previous way of creating the LRA was not working anymore, lraID was not initialized and therefore was causing a NullPointerException at https://github.com/jbosstm/narayana/blob/main/rts/lra/coordinator/src/main/java/io/narayana/lra/coordinator/domain/service/LRAService.java#L107. So changing how it is initialized.

Reply:
I will take a look next week (I'm on PTO today but wanted to get this PR reviewed asap).

Reply:
But don't let this comment block the PR.

Reply:
The constructor should only be used when loading an LRA from the ObjectStore. So after constructing the LRA the expectation is that it restore_state will be called so that the URI for the LRA gets initialised. Clearly the code is brittle so it would be nice to firm it up in this area - I'll raise a JBTM so that we don't loose site of it.

Reply:
Thanks Mike!